beacon-chain/rpc/prysm/v1alpha1/validator/blocks_test.go

```diff
@@ -70,176 +67,6 @@ func TestServer_StreamAltairBlocks_ContextCanceled(t *testing.T) {
 	exitRoutine <- true
 }
 
-func TestServer_StreamAltairBlocks_OnHeadUpdated(t *testing.T) {
```

Comment:
Does this test should be removed? Just removing SetCurrentSyncCommittee seemes to be okey.